samples/ipc/ipc_service/sample.yaml

```
@@ -0,0 +1,66 @@
+sample:
+    name: IPC Service integration test
+    description: IPC Service integration and efficiency test
+tests:
+  samples.subsys.ipc.ipc_service.icmsg_default:
```

```suggestion
  sample.subsys.ipc.ipc_service.icmsg_default:
```
we are unifying the naming conventions #7290 
Please apply to all tests in this yaml